data/json/items/gunmod/laser_gunmods.json

```
@@ -99,5 +100,97 @@
     "damage_modifier": { "damage_type": "heat", "amount": 10 },
     "energy_drain_multiplier": 2.0,
     "min_skills": [ [ "weapon", 4 ], [ "electronics", 5 ] ]
+  },
+  {
+    "id": "add_intake_fan",
+    "type": "GUNMOD",
+    "name": { "str": "additional intake fan" },
+    "description": "An additional fan that mounts to the radiator intake. It slightly increases the cooling performance of the gun.",
+    "weight": "600 g",
+    "volume": "270 ml",
+    "integral_volume": "150 ml",
+    "price": 84000,
+    "price_postapoc": 1000,
+    "install_time": "20 s",
+    "material": [ "steel", "plastic" ],
+    "symbol": ":",
+    "color": "light_gray",
+    "location": "radiator",
+    "mod_targets": [ "pistol", "rifle" ],
+    "cooling_value_modifier": 0.5
+  },
+  {
+    "id": "heat_dissipation_sheets",
+    "type": "GUNMOD",
+    "name": { "str": "heat dissipation sheets" },
+    "description": "A micro-textured sheet that is applied to the heat exchange portion of the radiator. It increases cooling performance in a format that depends on the gun's original cooling performance.",
+    "weight": "150 g",
+    "volume": "600 ml",
+    "integral_volume": "0 ml",
+    "price": 84000,
+    "price_postapoc": 1000,
+    "install_time": "7 m",
+    "material": [ "steel" ],
+    "symbol": ":",
+    "color": "light_gray",
+    "location": "radiator",
+    "mod_targets": [ "pistol", "rifle" ],
+    "cooling_value_multiplier": 1.5
+  },
+  {
+    "id": "add_heat_sink_for_small",
+    "type": "GUNMOD",
+    "name": { "str": "additional heat sink for small" },
+    "description": "This is a common type additional heat sink in the form of thin metal plates are evenly spaced. It increases the heat capacity. Although it is designed for small guns, it does not mean that it cannot be attached to other guns.",
+    "weight": "300 g",
+    "volume": "200 ml",
```

```suggestion
    "volume": "1750 ml",
```

50 extra heat capacity is a lot, this thing would need to be pretty sizeable.

---

I came up with this number from v29, where v29 was 1 liter and the overheat_threshold was 60.
Since I considered that parts such as the laser emitter, converging lens, and transistors are parts that should not store the generated heat, I determined that the heat capacity is mostly from the heat sink, and if I subtracted the aforementioned parts from 1L, the heat sink would probably be about 200ml.

Whereas v29 has an overheat_threshold of 60,
Isn't it unnatural that it would require 1750ml to add 50 to the overheat_threshold?